.gitignore

```
@@ -18,3 +18,9 @@ pytest.xml
 # Jupyter Notebook artifacts #
 ##############################
 .ipynb_checkpoints/
+.idea/workspace.xml
+.idea/inspectionProfiles/profiles_settings.xml
+.idea/misc.xml
+.idea/modules.xml
+.idea/PyDynamic.iml
+.idea/vcs.xml
```

Maybe just exclude everything from PyCharm?

```suggestion
.idea/
```